workbench-core/authentication/src/plugins/cognitoAuthenticationPlugin.test.ts

```diff
@@ -86,7 +88,10 @@ describe('CognitoAuthenticationPlugin tests', () => {
 
   describe('constructor tests', () => {
     it('should throw PluginConfigurationError when the user pool id is invalid. Must match "<region>_<some string>" format', () => {
-      const badUserPoolIdConfig = { ...cognitoPluginOptions, userPoolId: 'badId' };
+      const badUserPoolIdConfig = {
+        ...cognitoPluginOptions,
+        webUiAppClient: { ...cognitoPluginOptions.webUiAppClient, userPoolId: 'badId' }
```

Comment:
it looks like tests are going through the UI client. This is the client which should have `ALLOW_ADMIN_USER_PASSWORD_AUTH` disabled and should not function with integration tests.

Reply:
Updated.